src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsChannelMapping.java

```diff
@@ -36,6 +37,15 @@ public static List<HearingChannel> getHearingChannels(@Valid SscsCaseData caseDa
     }
 
     public static HearingChannel getHearingChannel(@Valid SscsCaseData caseData) {
+
+        if (caseData.getAdjournCaseTypeOfNextHearing() != null) {
+            return Arrays.stream(HearingChannel.values())
+                .filter(v -> caseData.getAdjournCaseTypeOfNextHearing().equalsIgnoreCase(v.getValueEn()))
+                .findFirst().orElse(null);
```

Comment:
When testing locally, this is immediately returning null. The log.info below never gets called.

Reply:
moved log info